src/os/ui/im/action/filteractionsui.js

```diff
@@ -283,8 +283,6 @@ os.ui.im.action.FilterActionsCtrl.prototype.launchExport = function() {
 os.ui.im.action.FilterActionsCtrl.prototype.launchImport = function() {
   os.metrics.Metrics.getInstance().updateMetric(os.im.action.Metrics.IMPORT, 1);
 
-  var event = new os.ui.im.ImportEvent(os.ui.im.ImportEventType.FILE, undefined, undefined, {
-    'layerId': this.entryType
-  });
+  var event = new os.ui.im.ImportEvent(os.ui.im.ImportEventType.FILE);
```

Comment:
We don't want to remove the layer ID context here. The layer ID provided here is passed to the filter import directives via this line: https://github.com/ngageoint/opensphere/blob/master/src/os/ui/filter/im/filterimportui.js#L58

The FilterImportCtrl (or in this case, FilterActionImportCtrl) uses that context to configure the UI a little differently. That's where the bug is occurring.